File: C++/freecodecamp_c++_course/23_classes_and_objects.cpp
#include <iostream>

using namespace std;

class Book {
  public:
    string title;
    string author;
    int amtPages;
};

int main()
{
  Book book1;
  book1.title = "Sup dude";
  book1.author = "Yo Momma";
  book1.amtPages = 20000;

  Book theHobbit;
  theHobbit.title = "The Hobbit";
  theHobbit.author = "J.R.R. Tolkien";
  theHobbit.amtPages = 800;

  cout << theHobbit.title << endl;
  
  return 0;
}

File: C#/freecodecamp_c#_course/23_classes_and_objects.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClassesAndObjects
{
  class Program
  {
    static void Main(string[] args)
    {
      Book book1 = new Book();
      book1.title = "Lord of the Rings";
      book1.author = "J.R.R. Tolkien";
      book1.numPages = 10000;

      Console.WriteLine(book1.title);

      Book playboyIssueOne = new Book();
      playboyIssueOne.title = "Playboy Issue #1";
      playboyIssueOne.author = "Hugh Hefner";
    }
  }
}
